src/main/java/de/tum/in/www1/artemis/domain/enumeration/AssessmentType.java

```diff
@@ -4,5 +4,5 @@
  * The AssessmentType enumeration.
  */
 public enum AssessmentType {
-    AUTOMATIC, SEMI_AUTOMATIC, MANUAL
+    AUTOMATIC, AUTOMATIC_ATHENA, SEMI_AUTOMATIC, MANUAL
```

Comment:
Why do we need this? Can there be other automatic feedback for modelling exercises that does not come from Athena?

Reply:
The "AUTOMATIC_ATHENA" enum entry is introduced in this PR https://github.com/ls1intum/Artemis/pull/8199 and was only also introduced here because otherwise this PR could no longer be deployed to the ma-schwind Testserver - for now, I can remove the value again I suppose. 🤔